### ISSUE TITLE
Make rateAccuracy a setting

### ISSUE THREAD

**Issue by akelge** [kontena/lens#7743]
**What would you like to be added**:
Currently when I configure metrics reading on a cluster I have 5 options plus autodetect. Each option has its own hardcoded value for `rateAccuracy`, I would like if this can become a setting parameter.

**Why is this needed**:
I use `kube-prometheus-stack` helm chart to deploy the full stack on K8S clusters, but I would like to change the `scrapeInterval` to 1 minute instead of the default 30 seconds. If I do so this breaks CPU reporting in Lens, cause the detected prometheus (operator) has a default `rateAccuracy` of 1 minute that makes some CPU metrics query report empty results.

**Environment you are Lens application on:**
- Kubernetes distribution: not relevant
- Desktop OS: not relevant

**Comment by zalejus** [comment]
Hello everyone,
January 26, 2024 and I am in exactly the same situation. Same problem - unable to change scraping interval in Prometheus without losing CPU usage in Lens panel. If you ask me it should be one of the basic functionalities.

**Comment by dark-brains** [comment]
Yes same issue not working CPU Usage metrics for example for Node
